### ISSUE TITLE
versionCode not increased?

### ISSUE THREAD

**Issue by IzzySoft** [Shingyx/BoomSwitch#18]
Looks like you forgot to [increase `versionCode`](https://github.com/Shingyx/BoomSwitch/blob/master/app/build.gradle#L28) with the last release (it has it on `10`, as did the previous release). This means, no update notifications were sent to those having your app installed. Certainly an oversight, but I thought to let you know just in case. Thanks for taking care!

**Comment by Shingyx** [comment]
The latest released version being 10 is correct - it was incremented here 8f214343e02224d85b7f55fb3ee6d3f8a24b14c5.

Is this related to https://github.com/Shingyx/BoomSwitch/issues/16, where I changed the APK in the GitHub release in-place?

**Comment by IzzySoft** [comment]
Not sure. I've just walked the files in my quarantine directory and found `com.github.shingyx.boomswitch_10.1696181537.apk` – which means that on September 30th my updater pulled an APK with `versionCode: 10` while one with that very same `versionCode` was already present. So I checked with your `build.gradle` here and it also has `10` – hence I reached out.

But now that you remind me of that one, I also checked the signatures – and indeed the one in quarantine is signed by a different certificate. So you seem to be right with the relation – apologies for the false alert then! I'll simply remove that file from quarantine so I do not stumble upon it again.
